misc/docker-ci/check.mk

```diff
@@ -36,9 +36,10 @@ ossl1.1.1:
 		TEST_ENV='$(TEST_ENV)'
 
 dtrace:
-	docker run $(DOCKER_RUN_OPTS) $(CONTAINER_NAME) \
+	docker run $(DOCKER_RUN_OPTS) h2oserver/h2o-ci:ubuntu2004  \
```

Comment:
Hard-coding ubuntu2004 since we never use ubuntu1604 for this scenario. On the other hand, `CMAKE_ARGS` may change in, for example, local tests.

Reply:
I'm not necessarily opposed to this change, though I'm not sure if I follow what you describe as the rationale. CMAKE_ARGS is hard-coded in other targets (e.g., `ossl-1.1+fuzz`).

If we are to make this kind of change, I would suggest hard-coding CMAKE_ARGS as well, and also changing the target name from `dtrace` to `dtrace+asan`.

Then, ci.yml would be purely a front-end for GitHub CI that sets concurrency paramaters.

Reply:
Right. I rather prefer that style. Fixed.